components/post_view/post_attachment_opengraph/post_attachment_opengraph.tsx

```diff
@@ -141,7 +141,7 @@ export default class PostAttachmentOpenGraph extends React.PureComponent<Props>
             props,
         };
 
-        return this.props.actions.editPost(patchedPost);
+        return this.props.actions?.editPost(patchedPost);
```

Comment:
`actions` should never be nullable in a redux-connected component.
What's likely happening is by importing `Props` from this component in the redux connector, the TS compiler is expecting all uses of the component to also pass in `actions` as a property to the redux-connected component.

You'll want to define a type in the redux connector called `OwnProps` that will only contain the props that the redux connector needs to run `mapStateToProps`.

Reply:
I'll see what I can do, thanks

Reply:
I followed your advice, thanks